File: diordnA_htiw/PracticeProject/android/app/src/main/java/com/practiceproject/LoginPage.java
package com.practiceproject;

import android.app.Activity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

public class LoginPage extends Activity {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        EditText Email=(EditText) findViewById(R.id.email);
        EditText Password=(EditText) findViewById(R.id.password);

        Button but=(Button) findViewById(R.id.button);

        but.setOnClickListener(view -> {
            //code
            String email=Email.getText().toString();
            System.out.println(email);
            String pass=Password.getText().toString();
            System.out.println(SignupPage.Email+" "+SignupPage.Password);
            if(email.equals(SignupPage.Email)&&pass.equals(SignupPage.Password))
            {
                Toast.makeText(getApplicationContext(),"Login Successful!",Toast.LENGTH_LONG).show();
            }
            else {
                Toast.makeText(getApplicationContext(),"Unsuccessful!",Toast.LENGTH_LONG).show();
            }
        });
    }
}

File: diordnA_htiw/PracticeProject/android/app/src/main/java/com/kotlineSampleProject/Loginpage.kt
//package com.practiceproject
//
//import android.app.Activity
//import android.os.Bundle
//import android.view.View
//import android.widget.Button
//import android.widget.EditText
//import android.widget.Toast
//
//class LoginPage : Activity() {
//    var SignupPage =SignupPage()
//    public override fun onCreate(savedInstanceState: Bundle?) {
//        super.onCreate(savedInstanceState)
//        setContentView(R.layout.activity_main)
//        val Email = findViewById<View>(R.id.email) as EditText
//        val Password = findViewById<View>(R.id.password) as EditText
//        val but = findViewById<View>(R.id.button) as Button
//        but.setOnClickListener { view: View? ->
//            //code
//            val email = Email.text.toString()
//            println(email)
//            val pass = Password.text.toString()
//
//            if (email == SignupPage.Email && pass == SignupPage.Password) {
//                Toast.makeText(applicationContext, "Login Successful!", Toast.LENGTH_LONG)
//                    .show()
//            } else {
//                Toast.makeText(applicationContext, "Unsuccessful!", Toast.LENGTH_LONG).show()
//            }
//        }
//    }
//}